src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java

```diff
@@ -107,7 +109,8 @@ private String firstOf(String[] array1, String[] array2) {
 	}
 
 	private String pathInfo() {
-		return ((javax.servlet.http.HttpServletRequest) webRequest.getNativeRequest()).getPathInfo();
+		HttpServletRequest request = (javax.servlet.http.HttpServletRequest) webRequest.getNativeRequest();
+		return request.getRequestURI().substring(request.getContextPath().length());
```

Comment:
please describe why using `getPathInfo` doesn't work for you? How can I reproduce it?

Reply:
I created this project https://github.com/kelvinsleonardo/demo-specification to reproduce the problem.